src/chart/sankey/SankeyView.ts

```diff
@@ -231,7 +231,9 @@ class SankeyView extends ChartView {
             const focus = emphasisModel.get('focus');
             toggleHoverEmphasis(
                 curve,
-                focus === 'adjacency' ? edge.getAdjacentDataIndices() : focus,
+                focus === 'adjacency' ? edge.getAdjacentDataIndices()
+                : focus === 'trajectory' ? edge.getFullPathDataIndices()
+                : focus,
```

Comment:
Code style preference

```suggestion
                focus === 'adjacency'
                    ? edge.getAdjacentDataIndices()
                    : focus === 'trajectory'
                        ? edge.getFullPathDataIndices()
                        : focus,
```